src/components/Accordion.tsx

```diff
@@ -0,0 +1,77 @@
+import { DummyData, Group } from "@/services/usos/getGroups";
+import { Accordion, AccordionContent, AccordionItem, AccordionTrigger } from "@/components/ui/accordion";
+
+type CombinedProps = {
+  index: number;
+  onClick: (index: number, group: Group) => void;
+} & DummyData;
+
+export default function GroupsAccordion({ index, onClick, ...props }: CombinedProps) {
+  //   return (
+  //     <div>
+  //       <h2>{index}</h2>
+  //       <h1>{props.registration.name}</h1>
+  //       <h2>
+  //         {props.courses.map((course) => {
+  //           return (
+  //             <div key={course.course.id}>
+  //               <div key={course.course.id}>{course.course.name}</div>
+  //               {course.groups.map((group) => {
+  //                 return (
+  //                   <div key={group.name}>
+  //                     <input type="checkbox" onClick={() => onClick(index, group)} />
+  //                     {group.name}
+  //                   </div>
+  //                 );
+  //               })}
+  //             </div>
+  //           );
+  //         })}
+  //       </h2>
+  //     </div>
+  //   );
```

Comment:
tbh usuwaj komentarze jak już dajesz pr'ke do review, to, że robisz prototyp nie oznacza, że można zostawiać śmieci w kodzie 😭